### PR TITLE
Add com.visualstudio.code.tool.git-lfs

### PR DESCRIPTION
**Pull request by ranisalt**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/git-lfs/git-lfs/issues/5285
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). **does not apply**
- [x] Any additional patches or files have been submitted to the upstream projects concerned. **does not apply**

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by ranisalt** [comment]
Hello! Adding this tool that is pretty much ubiquitous for repositories with binary content. Currently, I'd have to install the golang SDK, which is huge (460 MB+) to then install `git-lfs` using `go install`. I do not code in Go, so I would uninstall the SDK immediately after. Alternatively, I have to install prebuilt binaries, but I'd rather have it from source.

**Comment by hfiguiere** [comment]
bot, build com.visualstudio.code.tool.git-lfs

**Comment by hfiguiere** [comment]
Extensions need to skip the icon check.

**Comment by ranisalt** [comment]
@hfiguiere added skip icon check to `flathub.json`

**Comment by hfiguiere** [comment]
bot, build com.visualstudio.code.tool.git-lfs

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "skip-icons-check": true,
```
Comment (hfiguiere) [inline flathub.json:2]:
```suggestion
    "skip-icons-check": true
```

**Comment by ranisalt** [comment]
bot, build com.visualstudio.code.tool.git-lfs

**Comment by lu9dce** [comment]
ohh!!! tks

**Comment by noonsleeper** [comment]
Hi @hfiguiere, can you give us an update on this?
This tool can help 3 projects at the same time code, code-oss and codium

File: com.visualstudio.code.tool.git-lfs.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+id: com.visualstudio.code.tool.git-lfs
+branch: '22.08'
+build-extension: true
+sdk: org.freedesktop.Sdk//22.08
```
Comment (bbhtt) [inline com.visualstudio.code.tool.git-lfs.yml:4]:
```suggestion
branch: '23.08'
build-extension: true
sdk: org.freedesktop.Sdk//23.08
```

Comment (hfiguiere) [reply com.visualstudio.code.tool.git-lfs.yml:4]: NO.

The host app is still 22.08

Comment (bbhtt) [reply com.visualstudio.code.tool.git-lfs.yml:4]: Hmm, thought they updated to 22.08 by now. Ignore.

**Pull request closed without merging**